doc/progress.rst

```diff
@@ -20,6 +20,8 @@ Changelog
   OpenML.
 * ADD #564: New helpers to access the structure of a flow (and find its
   subflows).
+* ADD #618: The software will from now on retry to connect to the server if a
+  connection failed. The number of retries can be configured.
```

Comment:
can you also add #602 please? maybe close to columntransformer (sorry i forgot)

Reply:
As the column transformer is only added in this new release it makes no sense to also add a fix to this new features in the change log.